app/models/stop_condition.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class StopCondition < ApplicationRecord
+  belongs_to :extraction_definition
+
+  # rubocop:disable Lint/UnusedBlockArgument
+  # rubocop:disable Security/Eval
+  def evaluate(document)
+    block = ->(response) { eval(content) }
+
+    block.call(document)
+  rescue StandardError => e
+    e
```

Comment:
do we need to do something with `e`?

Reply:
Not at the moment. We don't have any error reporting yet but once we do we will populate the report with errors :).